### ISSUE TITLE
When encoding a JSON variable in the BODY of an API (POST) CALL there is a mistake with extras "" 

### ISSUE THREAD

**Issue by AirtonLucianoAragao** [FlutterFlow/flutterflow-issues#3686]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When encoding a JSON variable in the BODY of an API (POST) CALL there is a mistake with extras "" 

### Expected Behavior

I expect a valid JSON encoding

### Steps to Reproduce

1. Define an API CALL type POST
2. SET an variable as JSON TYPE and LIST = TRUE
3. Go to Response & Test tab
4. Insert a list of JSON objects inside [] like [{"b":"AA"},{"b":"CC"}]
5. See BODY formated as "["{b: AA}","{b: CC}"]"
6. 

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4oz+fl4ot2xNtJ7qqNc8JViSQUH1t+U5EnkcpOGDsdF4iuPOxzaeLCTFFvcMe6TWw2KEWGgmIEwPDqvvPTKsBdHxedR4hfyMxTWDnxfz6/aovXCs+sZHdtN95jN0yv06urgBJTBLZqLl472U2YNvCQQj3rN5f4Zwh9f6PHaOI=

### Visual documentation

![Captura de tela de 2024-08-25 15-51-39](https://github.com/user-attachments/assets/4e973c59-1a54-4e71-a902-fd6501f90d41)
![Captura de tela de 2024-08-25 15-52-18](https://github.com/user-attachments/assets/65995cf6-bbd2-4f72-8354-4996f3996a32)
![Captura de tela de 2024-08-25 15-52-37](https://github.com/user-attachments/assets/5d281640-ed67-4ebe-b772-cf1d81702242)

### Environment

```markdown
- FlutterFlow version: Latest
- Platform: Ubuntu
- Browser name and version: Crhome
- Operating system and version affected:
```

**Comment by AirtonLucianoAragao** [comment]
Now I noticed that the problem is that FF doesn't insert the \ (escape slash) in order to properly configurate the JSON provided by our code (or directly, when testing the API).

**Comment by Alezanello** [comment]
Hello!!<br><br>I Couldn't replicate your issue, is sending perfectly in the test i made:

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/3186a7bb-20ac-49d6-b0ab-0f16fc042ea0/809f4e11-bdf2-4794-921f-a17511f6e0d9?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS8zMTg2YTdiYi0yMGFjLTQ5ZDYtYjBhYi0wZjE2ZmMwNDJlYTAvODA5ZjRlMTEtYmRmMi00Nzk0LTkyMWYtYTE3NTExZjZlMGQ5IiwiaWF0IjoxNzI0NzY3NjQxLCJleHAiOjMzMjk1MzI3NjQxfQ.P4K_87NYAJTUoR5MzE5InVLuX-2TS62PVvFfhgyXRM0 " alt="image.png" width="1073" height="465" />

<br>This is the public URL of the project if you want to see the setup.<br>[https://app.flutterflow.io/project/qa-1814-e5fh83](https://app.flutterflow.io/project/qa-1814-e5fh83)

**Comment by AirtonLucianoAragao** [comment]
Hi Alezanello.

Looking to your project I noticed that at my BODY in CALL DEFINITION there were quotes surrounding the JSON variable and this is not true at your definition. This occured maybe because I changed this variable from STRING to JSON.

Thanks a lot and sorry for my mistake.
![Captura de tela de 2024-08-27 14-24-38](https://github.com/user-attachments/assets/1de8edcd-4d8a-4fbd-b2cc-44a3faeec1b9)
